source/agora/node/WasmEngine.d

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
+
+    Contains the wasmer WebAssembly engine to be used for Trust Contracts
```

Comment:
How about simply explaining what `Engine`, `Store`, and `Module` are for in this part?

Reply:
I added a few comments and a link to a relevant website.